docs/data/stashcache/run-stash-origin-container.md

```diff
@@ -27,11 +27,13 @@ Before starting the installation process, consider the following points:
 
 1. **Docker:** For the purpose of this guide, the host must have a running docker service and you must have the ability
 to start containers (i.e., belong to the `docker` Unix group).
-1. **Network ports:** The origin listens for incoming HTTP/S and XRootD connections on ports 1094 and 1095 (by
-default).
+1. **Network ports:** The origin listens for incoming HTTP(S) and XRootD connections on port 1094.
 1. **File Systems:** The origin needs a host partition to store user data.
+1. **Hardware requirements:** We recommend that an origin has at least 1Gbps connectivity and 8GB of RAM.
+1. **Host certificate:** Required for joining the OSDF.
```

Comment:
```suggestion
1. **Host certificate:** Presented to origin service clients to prove authenticity.
```

I think this goes without saying being under what's effectively a requirements section so maybe we should talk about why it's necessary?